operations/iuf/workflows/admin_directory.md

```diff
@@ -0,0 +1,95 @@
+    cd ${ADMIN_DIR}
+    ```
+
+1. Copy the `sat bootprep` and `product_vars.yaml` files from the uncompressed HPC CSM Software Recipe distribution file in the media directory to the current directory.
+
+    (`ncn-m001#`) Copy `sat bootprep` and `product_vars.yaml` files
+
+    ```bash
+    cp "${MEDIA_DIR}"/hpc-csm-software-recipe-*/vcs/product_vars.yaml .
+    cp -r "${MEDIA_DIR}"/hpc-csm-software-recipe-*/vcs/bootprep .
+    ```
+
+    (`ncn-m001#`) Examine the contents of `${ADMIN_DIR}` to verify the expected content is present
+
+    ```bash
+    find . -type f
+    ```
+
+    Example output:
+
+    ```text
+    ./bootprep/management-bootprep.yaml
+    ./bootprep/compute-and-uan-bootprep.yaml
+    ./product_vars.yaml
+    ```
+
+1. Edit the `compute-and-uan-bootprep.yaml` and `management-bootprep.yaml` files to account for any site deviations from the default values. For example:
+    - Comment out the `slurm-site` CFS configuration layer and uncomment the `pbs-site` CFS configuration layer in `compute-and-uan-bootprep.yaml` if PBS is the preferred workload manager
+    - Uncomment the `gpu-{{recipe.version}}` CFS configuration layer and `gpu-image` image definition in `compute-and-uan-bootprep.yaml` if the system has GPU hardware
+    - Comment out any CFS configuration layers in `compute-and-uan-bootprep.yaml` and `management-bootprep.yaml` files for products that are not needed on the system
+    - Any other changes needed to reflect site preferences
+
+1. Create a `site_vars.yaml` file in `${ADMIN_DIR}`. This file will contain key/value pairs for any configuration changes that should override entries in the `default` section of the HPE-provided `product_vars.yaml` file.
+   There are comments at the top of the `product_vars.yaml` file that describe the variables and related details. The following are a few examples of `site_vars.yaml` changes:
+    - Add a `default` section containing a `network_type: "cassini"` entry to designate that Cassini is the desired Slingshot network type to be used when executing CFS configurations later in the workflow
+    - Add a `suffix` entry to the `default` section to append a string to the names of CFS configuration, image, and BOS session template artifacts created during the workflow to make them easy to identify
+
+   Additional information on `site_vars.yaml` files can be found in the [Site and recipe variables](../IUF.md#site-and-recipe-variables) and [`update-vcs-config`](../stages/update_vcs_config.md) sections.
+
+    1. Create a `site_vars.yaml` file with desired key/value pairs
+
+    2. Ensure the `site_vars.yaml` file contents are formatted correctly. The following text is an example for verification purposes only.
+
+       (`ncn-m001#`) Display the contents of an **example** `site_vars.yaml` file
+
+       ```bash
+       cat site_vars.yaml
+       ```
+
+       Example output:
+
+       ```text
+       default:
+         network_type: "cassini"
+         suffix: "-test01"
+       uss:
```

Comment:
We need to remove lines 70-72 uss site_vars.yaml this was added in CSM 1.6

https://github.com/Cray-HPE/docs-csm/pull/5459

https://github.com/Cray-HPE/docs-csm/blob/release/1.5/operations/iuf/workflows/configuration.md?plain=1#L77

Reply:
Done the required changes